### PR TITLE
fix adding invalid emails that exist in the DB

### PR DESCRIPTION
**Pull request by DanielPalafox**
https://trello.com/c/V4U3RzDU/7015-qa-adding-invalid-emails-that-exist-in-the-db-at-my-orcid-account

### CODE REVIEW

**Comment by DanielPalafox** [comment]
[[QA] Adding invalid emails that exist in the DB at /my-orcid & /account](https://trello.com/c/V4U3RzDU/7015-qa-adding-invalid-emails-that-exist-in-the-db-at-my-orcid-account)

File: orcid-web/src/main/webapp/static/javascript/ng1Orcid/app/modules/emailsForm/emailsForm.component.ts [hunk 1]
```diff
@@ -224,6 +224,11 @@ export class EmailsFormComponent implements AfterViewInit, OnDestroy, OnInit {
 
                     if (this.inputEmail.errors.length == 0) {
                         this.initInputEmail();
+                    } else {
+                        if (this.inputEmail.errors.filter(error => error.startsWith('Please enter your email address in the').length > 0)
```
Comment (amontenegro) [inline orcid-web/src/main/webapp/static/javascript/ng1Orcid/app/modules/emailsForm/emailsForm.component.ts:228]: @DanielPalafox this is not going to work for non english error messages right?

**Review by amontenegro — CHANGES_REQUESTED** [review]

File: orcid-web/src/main/webapp/static/javascript/ng1Orcid/app/modules/emailsForm/emailsForm.component.ts (hunk 1 above)
Comment (amontenegro) [reply orcid-web/src/main/webapp/static/javascript/ng1Orcid/app/modules/emailsForm/emailsForm.component.ts:228]: @DanielPalafox ping! have you found a different solution for this?

Comment (DanielPalafox) [reply orcid-web/src/main/webapp/static/javascript/ng1Orcid/app/modules/emailsForm/emailsForm.component.ts]: @amontenegro I fixed it looking an email which is consistent in the error message in different languages

**Pull request merged**